spectator-ext-jvm/src/main/java/com/netflix/spectator/jvm/Jmx.java

```diff
@@ -16,11 +16,10 @@
 package com.netflix.spectator.jvm;
 
 import com.netflix.spectator.api.Registry;
+import com.netflix.spectator.api.patterns.PolledMeter;
 import com.typesafe.config.Config;
 
-import java.lang.management.BufferPoolMXBean;
-import java.lang.management.ManagementFactory;
-import java.lang.management.MemoryPoolMXBean;
+import java.lang.management.*;
```

Comment:
We typically try to avoid wildcard imports

Reply:
Alright, will make those explicit.